ripe/atlas/cousteau/stream.py

```diff
@@ -27,13 +27,21 @@ class AtlasStream(object):
         "error": CHANNEL_ERROR,
     }
 
-    def __init__(self):
+    def __init__(self, log_errors=True):
         """Initialize stream"""
 
         self.iosocket_server = "atlas-stream.ripe.net"
         self.iosocket_resource = "/stream/socket.io"
 
         self.socketIO = None
+        self.log_errors = log_errors
+        self.error_callback = None
+
+    def report_errors(self, error):
+        if self.error_callback:
+            self.error_callback(error)
+        elif self.log_errors:
+            print(error)
```

Comment:
I think that if "log_errors" is True, then we should log errors regardless of whether "on_error()" has also been called. That way it's explicit, and the user can always set "log_errors" to False if they don't want logging.